src/app/core/services/phase.service.ts

```diff
@@ -78,11 +79,14 @@ export class PhaseService {
     }
     if (this.currentPhaseUrl == null) {
       this.errorHandlingService.handleGeneralError('Repo is not ready');
+      return ('not accessible');
     } else {
+      copyUrl = this.currentPhaseUrl;
+      this.currentPhaseUrl = null;
```

Comment:
Any reason why you make currentPhaseUrl `null`?

Reply:
Yes, there was a bug where if the user first logs in, then logs out and tries to log in again, it uses the old value of currentPhaseUrl

Reply:
Then when the user logs out, we should reset the value of currentPhaseUrl.

Reply:
I will make another PR to resolve this.